smartsim/_core/_cli/validate.py

```diff
@@ -52,6 +50,7 @@
 # pylint: disable=import-error,import-outside-toplevel
 # mypy: disable-error-code="import"
 
+# TODO add a new Client
```

Comment:
If you want, `smart validate` is almost certainly not going to set up a redis instance in SSv1.0. I would basically keep the utilites in this file setting up a temporary directory, a working env, etc, but just remove everything else that is even slightly SSv0.7 API like (e.g. remove `test_install`, `_make_managed_local_feature_store`, `_test_tf_install`, `_test_torch_install`, and `_test_onnx_install`). That should save you from needing to make a `smartredis.Client` mock!

Reply:
removed